packages/ui/src/utils/UiHelperUtil.ts

```diff
@@ -119,5 +119,28 @@ export const UiHelperUtil = {
     const roundedNumber = Math.abs(number) >= threshold ? Number(number.toFixed(fixed)) : number
 
     return roundedNumber
+  },
+  /**
+   * Format the given number or string to human readable numbers with the given number of decimals
+   * @param value - The value to format. It could be a number or string. If it's a string, it will be parsed to a float then formatted.
+   * @param decimals - number of decimals after dot
+   * @returns
+   */
+  formatNumberToLocalString(value: string | number | undefined, decimals = 2) {
+    if (value === undefined) {
+      return '0.00'
+    }
+
+    if (typeof value === 'number') {
+      return value.toLocaleString('en-US', {
+        maximumFractionDigits: decimals,
+        minimumFractionDigits: decimals
+      })
+    }
+
+    return parseFloat(value).toLocaleString('en-US', {
+      maximumFractionDigits: decimals,
+      minimumFractionDigits: decimals
+    })
```

Comment:
isnt this repeated in `NumberUtils`?

Reply:
It should be removed, let me check

Reply:
Good catch, we have it more than one, removed the others!